package.json

```diff
@@ -111,8 +112,6 @@
     "react-native-gesture-handler": "~2.16.1",
     "react-native-get-random-values": "~1.11.0",
     "react-native-gifted-chat": "^2.0.1",
-    "react-native-map-clustering": "^3.4.2",
-    "react-native-maps": "1.14.0",
```

Comment:
with removing this package, the following occurs. please adjust it:

```
$ patch-package
patch-package 8.0.0
Applying patches...
Error: Patch file found for package react-native-maps which is not present at node_modules/react-native-maps
---
patch-package finished with 1 error(s).
```

Reply:
adjusted it:
```
$ patch-package
patch-package 8.0.0
Applying patches...
No patch files found
```

should I remove the `patch-package 8.0.0` completely?

Reply:
if we do not use/need it, yes.

Reply:
removed it entirely.